internal/environment/template-environment-job.hcl

```diff
@@ -11,23 +11,24 @@ job "template-0" {
       sticky  = false
     }
     count = 1
-    scaling {
```

Comment:
This seems to be a historical artifact from the time we had multiple tasks per job. The [documentation](https://developer.hashicorp.com/nomad/docs/job-specification/scaling) does not point out a benefit for our current use.

Reply:
Thanks for identifying that, I am happy to remove it.